### ISSUE TITLE
Send message when multicore == TRUE and control$loss = "calFast"

### ISSUE THREAD

**Issue by mfasiolo** [mfasiolo/qgam#27]
We need to tell users that multicore is useless when using the sandwich calibration
